docs/devenv_cloud.md

```diff
@@ -0,0 +1,159 @@
+* For the `aarch64` architecture, one of the [c6g instances](https://aws.amazon.com/ec2/instance-types/c6g/) (i.e. `c6g.metal`)
+
+For development-only host setup, it is recommended to use the following instances **without** hardware virtualization support:
+* For the `x86_64` architecture, [c5.2xlarge](https://aws.amazon.com/ec2/instance-types/c5/) instance
+* For the `aarch64` architecture, [c7g.2xlarge](https://aws.amazon.com/ec2/instance-types/c7g/) instance
+
+> The Europe (Ireland) `eu-west-1` or US East (N. Virginia) `us-east-1` regions are recommended as they usually have the widest selection of hardware options.
+
+Review and configure the following VPC resources.
+* A default VPC settings should be sufficient
+* A default subnet in one of the availability zones with access to the Internet
+
+Review and configure the following EC2 resources.
+* A default security group with `Inbound rules` set to **only** allow SSH traffic on port 22. It is recommended to further restrict access to the SSH port from your IP address.
+* Import your `Key Pair` used for accessing the virtual instances
+
+Proceed by creating a virtual instance using the following parameters.
+* Use a descriptive name prefixed by your user name if the account is shared with others (i.e. `myuser-ushift-dev`)
+* Select the Red Hat Linux operating system family
+* Select the `64-bit (x86)` or `64 bit (Arm)` architecture
+* Select Red Hat Enterprise Linux 8 (HVM) operating system. 
+> Do not attempt to use RHEL 9 as it is not supported for the development environment. The operating system will be upgraded to the 8.7 version in the configuration stage.
+* Select the instance type depending on your architecture of choice:
+    * e.g. `c5.metal` for `x86_64`
+    * e.g. `c6g.metal` for `aarch64`
+* Select your `Key pair`
+* Select your security group, **only allowing** SSH port 22 inbound traffic
+* Configure storage with the following volumes:
+    * 90 GiB root volume on `gp3` storage type
+    * 10 GiB data volume on `gp3` storage type
+
+Review your selections and lauch the virtual instance. 
+> Unless noted otherwise, all the further configuration should be performed on the cloud virtual instance created in this section.
+
+### Other Cloud Environments
+
+Other public cloud vendors and environments should also be suitable for running the MicroShift development environment, provided the appropriate hardware and operating system configuration is available.
+
+## Development Virtual Machine
+
+Log into the virtual instance running in the cloud using the appropriate login credential (i.e. `ec2-user` for AWS) and a private SSH key.
+
+```bash
+ssh -i ~/.ssh/<private_key> ec2-user@<instance_public_IP>
+```
+
+Create and configure the `microshift` user account, also allowing SSH logins using a password.
+
+```bash
+PASSWORD="<strong password>"
+
+sudo dnf install -y perl
+sudo useradd -m -s /bin/bash -p $(perl -e"print crypt('${PASSWORD}', 'microshift')") microshift
```

Comment:
Use `openssl passwd -6 "${PASSWORD}"` instead, which also allows you to avoid installing perl.